### ISSUE TITLE
Add feature to import affiliates via CSV

### ISSUE THREAD

**Issue by wordmaid** [AffiliateWP/AffiliateWP#2137]
Importing affiliates - could we do this? A customer has asked if it would be possible to import affiliates via CSV into the WordPress admin. If they had all the required fields, would it be possible?

**Comment by pippinsplugins** [comment]
We will be able to soon after #338 is completed.

**Comment by wordmaid** [comment]
Another request: https://secure.helpscout.net/conversation/343587113/56643/?folderId=205524

**Comment by DrewAPicture** [comment]
FYI, the PR on #338 actually implements affiliate imports as a proof of concept, despite the issue title ;)

**Comment by DrewAPicture** [comment]
Closing this in favor of handling both importers via #338.
